src/services/oracle.ts

```diff
@@ -90,6 +90,21 @@ export class OracleService<T extends ChainId> extends ContractService<T> {
     return await this.contract.read.getNormalizedValueUsdc(token, amount, overrides).then(int);
   }
 
+  /**
+   * Get the normalized Usdc value for a list of tokens and corresponding quantities.
+   * @param tokens
+   * @param amounts
+   * @param overrides
+   * @returns Usdc exchange rate (6 decimals)
+   */
+  async getNormalizedValuesUsdc(tokens: Address[], amounts: Integer[], overrides: CallOverrides = {}): Promise<Usdc[]> {
```

Comment:
This is great, and is a useful addition. I think [the issue](https://github.com/yearn/yearn-sdk/issues/57) is referring to using an on-chain multicall contract so calls to the web3 provider can be reduced if you wanted to have a go at implementing that and using it here?

Reply:
Sure.  I'll give it a shot and post updates.

Reply:
@jstashh multicall should work now with the indexed finance multicall lib, still need to figure out how to use the CallOverrides, getting the following error

```
types/values length mismatch (count={"types":2,"values":3}, value={"types":[{"name":null,"type":"address","indexed":null,"components":null,"arrayLength":null,"arrayChildren":null,"baseType":"address","_isParamType":true},{"name":null,"type":"uint256","indexed":null,"compon
ents":null,"arrayLength":null,"arrayChildren":null,"baseType":"uint256","_isParamType":true}],"values":["0x0bc529c00C6401aEF6D220BE8C6Ea1667F6Ad93e","1000000000000000000",{}]}, code=INVALID_ARGUMENT, version=abi/5.5.0)
```

once this is resolved, I can lint/prettify and add more comprehensive tests.